Incorrect file extension in "exports" (package.json)

**Describe the bug**
There are 2 references to `"./dist/vue-qrcode-reader.umd.cjs"` in the `package.json`. But that file does not exist in the `dist` directory. This is causing eslint to throw errors relating to not being able to find an import.

**To Reproduce**
`yarn install vue-qrcode-reader`

Attempt to import something, e.g.: `import { QrcodeStream } from 'vue-qrcode-reader'`

Run eslint on project and hit error:
```
1:1   error  Resolve error: Cannot find module '/the/path/to/node_modules/vue-qrcode-reader/dist/vue-qrcode-reader.umd.cjs'
```

Note that the import works fine when running our app (using vite to build) but eslint fails.

--- a/package.json
+++ b/package.json
@@ -24,11 +24,11 @@
   },
   "main": "./dist/vue-qrcode-reader.js",
   "module": "./dist/vue-qrcode-reader.js",
-  "unpkg": "./dist/vue-qrcode-reader.umd.cjs",
+  "unpkg": "./dist/vue-qrcode-reader.umd.js",
   "exports": {
     ".": {
       "import": "./dist/vue-qrcode-reader.js",
-      "require": "./dist/vue-qrcode-reader.umd.cjs",
+      "require": "./dist/vue-qrcode-reader.umd.js",
       "types": "./dist/index.d.ts"
     }
   },

